.eslintrc

```diff
@@ -64,5 +69,35 @@
         ]
       }
     ]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/e2e/**"],
+      "rules": {
+        "testing-library/await-async-events": "off",
+        "testing-library/await-async-query": "off",
+        "testing-library/await-async-utils": "off",
+        "testing-library/no-await-sync-events": "off",
+        "testing-library/no-await-sync-queries": "off",
+        "testing-library/no-container": "off",
+        "testing-library/no-debugging-utils": "off",
+        "testing-library/no-dom-import": "off",
+        "testing-library/no-global-regexp-flag-in-query": "off",
+        "testing-library/no-manual-cleanup": "off",
+        "testing-library/no-node-access": "off",
+        "testing-library/no-promise-in-fire-event": "off",
+        "testing-library/no-render-in-lifecycle": "off",
+        "testing-library/no-unnecessary-act": "off",
+        "testing-library/no-wait-for-multiple-assertions": "off",
+        "testing-library/no-wait-for-side-effects": "off",
+        "testing-library/no-wait-for-snapshot": "off",
+        "testing-library/prefer-find-by": "off",
+        "testing-library/prefer-implicit-assert": "off",
+        "testing-library/prefer-presence-queries": "off",
+        "testing-library/prefer-query-by-disappearance": "off",
+        "testing-library/prefer-screen-queries": "off",
+        "testing-library/render-result-naming-convention": "off"
+      }
+    }
+  ]
```

Comment:
@jayasanka-sack @denniskigen I manually disabled the rules individually, Got the list of rules form their [GitHub](https://github.com/testing-library/eslint-plugin-testing-library?tab=readme-ov-file#supported-rules) and excluded the ones that have not been enabled for React

Reply:
@denniskigen what do you think about this solution?

Reply:
Fine with me!